datafusion/core/src/physical_plan/file_format/parquet/page_filter.rs

```diff
@@ -421,7 +423,16 @@ macro_rules! get_min_max_values_for_page_index {
                     vec.iter().map(|x| x.$func().cloned()),
                 )))
             }
-            Index::INT96(_) | Index::BYTE_ARRAY(_) | Index::FIXED_LEN_BYTE_ARRAY(_) => {
+            Index::BYTE_ARRAY(index) => {
+                let vec = &index.indexes;
+                let array: StringArray = vec
```

Comment:
I am not 100% sure if this is ok (like what if the parquet data got mapped to a LargeStringArray? 🤔

Reply:
we need check the logical type for the value.
BYTE_ARRAY in the parquet can represent many logical types, such as `DECIMAL`

Reply:
should also support the type `BYTE_ARRAY` in the `null_counts` of `PagesPruningStatistics`